gt/main.c

```diff
@@ -936,6 +1111,10 @@ config_gt_instance(struct gt_config *gt_conf, unsigned int lcore_id)
 	int ret;
 	char lua_entry_path[128];
 	unsigned int block_idx = get_block_idx(gt_conf, lcore_id);
+
+	/* Maximum TTL in cycles for each fragmented packet. */
+	uint64_t frag_cycles = (rte_get_tsc_hz() + MS_PER_S - 1) /
+		MS_PER_S * gt_conf->frag_max_flow_ttl_ms;
```

Comment:
Did you copy this expression from somewhere? It seems wrong to me since the units don't match. Can you explain it?

By the way, our `main/main.c:time_resolution_init()` sets the global variable `cycles_per_sec`, which is equivalent to `rte_get_tsc_hz()`. Could check if `rte_get_tsc_hz()` is more precise than our code? If so, we can simplify `main/main.c:time_resolution_init()`. If not, we have one more contribution to DPDK.

Reply:
Yes, I copied the expression from the example code: http://dpdk.org/doc/api/examples_2ip_reassembly_2main_8c-example.html#a66

Since ``rte_get_tsc_hz()`` can return the cycles per second, ``frag_cycles`` is the results of the cycles for ``gt_conf->frag_max_flow_ttl_ms`` milliseconds.

I think our solution is equivalent to ``rte_get_tsc_hz()``, since both are measured using the processor's Time Stamp Counters.

Reply:
The expression is correct. I miscalculated the units the first time.

I opened issue #82 to deal with `time_resolution_init()` in a future pull request.